src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -235,6 +235,10 @@ void CoreWorkerDirectTaskSubmitter::RequestNewWorkerIfNeeded(
           // A local request failed. This shouldn't happen if the raylet is still alive
           // and we don't currently handle raylet failures, so treat it as a fatal
           // error.
+          RAY_LOG(ERROR)
+              << "Core worker failed to get responses from local raylet. This is most "
```

Comment:
```suggestion
              << "The worker failed to receive a response from the local raylet. This is most "
```

(Updating because "core worker" doesn't mean anything to a user)